kapitan/inputs/helm.py

```diff
@@ -0,0 +1,123 @@
+        # save the template output to temp dir first
+        error_message = self.render_chart(
+            chart_dir=file_path,
+            output_path=temp_dir,
+            helm_values_file=self.helm_values_file,
+            helm_values_files=self.helm_values_files,
+            **self.helm_params,
+        )
+        if error_message:
+            raise HelmTemplateError(error_message)
+
+        walk_root_files = os.walk(temp_dir)
+        for current_dir, _, files in walk_root_files:
+            for file in files:  # go through all the template files
+                rel_dir = os.path.relpath(current_dir, temp_dir)
+                rel_file_name = os.path.join(rel_dir, file)
+                full_file_name = os.path.join(current_dir, file)
+                with open(full_file_name, "r") as f:
+                    item_path = os.path.join(compile_path, rel_file_name)
+                    os.makedirs(os.path.dirname(item_path), exist_ok=True)
+                    with CompiledFile(
+                        item_path,
+                        self.ref_controller,
+                        mode="w",
+                        reveal=reveal,
+                        target_name=target_name,
+                    ) as fp:
+                        yml_obj = list(yaml.safe_load_all(f))
+                        fp.write_yaml(yml_obj)
+                        logger.debug("Wrote file %s to %s", full_file_name, item_path)
+
+        self.helm_values_file = None  # reset this
+        self.helm_params = {}
+        self.helm_values_files = []
+
+    def default_output_type(self):
+        return None
+
+    def render_chart(self, chart_dir, output_path, **kwargs):
+        args = ["template"]
+        """renders helm chart located at chart_dir, and stores the output to output_path"""
+        if kwargs.get("helm_values_files", []):
+            for file_name in kwargs["helm_values_files"]:
+                args.append("-f")
+                args.append(file_name)
+
+        if kwargs.get("helm_values_file", None):
+            args.append("-f")
+            args.append(kwargs["helm_values_file"])
+
+        if kwargs.get("namespace", None):
+            args.append("-n")
+            args.append(kwargs["namespace"])
+
+        args.append("--output-dir")
+        args.append(output_path)
+
+        if self.kube_version:
+            args.append("--api-versions")
+            args.append(self.kube_version)
+
+        if kwargs.get("validate", False):
+            args.append("--validate")
+
+        if kwargs.get("name_template", None):
+            args.append("--name-template")
+            args.append(kwargs["name_template"])
+        else:
+            args.append(kwargs.get("release_name", "--generate-name"))
+
+        # uses absolute path to make sure helm interpret it as a
+        # local dir and not a chart_name that it should download.
+        args.append(os.path.abspath(chart_dir))
+
```

Comment:
Should we add `--include-crds` or make it configurable?

Reply:
And probably `--skip-tests`?

Reply:
Yes, this is both 2 reasonable flags to add by defaults. 
Should we add `skip_crds` and `include_tests` configuration flags then.

The only drawback I see with this names, is that helm already has an unrelated `--skip-crds` flags.

Reply:
I'd suggest to add `include_crds` and `skip_tests` just with a default of `true` to name them after the flags.